module-os/memory/UserMemStatsLogger.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <log/log.hpp>
+
+#include "usermem.h"
+
+struct UserMemStatsLogger
+{
+#if DEBUG_HEAP_ALLOCATIONS == 1
+    UserMemStatsLogger()
+    {
+        usermemResetStatistics();
+        fh1 = usermemGetFreeHeapSize();
+    }
+
+    ~UserMemStatsLogger()
+    {
+        size_t fh2 = usermemGetFreeHeapSize();
+        size_t ac = usermemGetAllocationsCount();
+        size_t dc = usermemGetDeallocationsCount();
+        size_t ami = usermemGetAllocatedMin();
+        size_t ama = usermemGetAllocatedMax();
+        size_t as = usermemGetAllocatedSum();
```

Comment:
I think that you can write more explicitly those variable names, as you did in usermem.c :P

Reply:
done :)